src/main/webapp/script.js

```diff
@@ -115,14 +115,23 @@ function clearSavedProfileStatus() {
   profileStatusElement.style.display = "none";
 }
 
-/** Gets recipe id list from query string */
+
+function getRecipe(){
+  /** Function gets recipe information from user input ID and displays the title on the page */
+  var numRecipe = document.getElementById("num-recipe").value;
+  fetch('/recipeInfo?numRecipe='+numRecipe).then(response => response.json()).then((recipeInfo) => {
```

Comment:
Are the enclosing parenthesis around "recipeInfo" necessary?

Reply:
I believe so